### ISSUE TITLE
In some cases, unsupported characters may be produced by openssl

### ISSUE THREAD

**Issue by martenjacobs** [martenjacobs/ToonRooter#13]
https://github.com/martenjacobs/ToonRooter/blob/af27a1c8c58f9f8bebc9f57f9a1ab30d65594490/payload/patch_toon.sh#L17
```
DEBUG:rooter:/ #
DEBUG:rooter:sh payload/patch_toon.sh "BLUFSLYD"
INFO:rooter:>>> Enabling root user. Your root password is: BLUFSLYD
DEBUG:rooter:sed: bad option in substitution expression
INFO:rooter:Cleaning up
DEBUG:rooter:rm -r payload
```

**Comment by martenjacobs** [comment]
`$PASSENC` may contain `/` characters, breaking the subsequent sed-commands
these should be escaped

**Comment by IgorYbema** [comment]
The characters in "salt" and "encrypted" are drawn from the set [a-zA-Z0-9./]. 
So it is safe to change the sed delimitors from / to for example #

**Comment by IgorYbema** [comment]
fixed in #14
